WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -450,13 +453,42 @@ extension MainTabBarController {
                     return
                 }
 
-                DispatchQueue.main.asyncAfter(deadline: .now() + Constants.screenTransitionsDelay) {
+                DispatchQueue.main.asyncAfter(deadline: .now() + Constants.blazeScreenTransitionsDelay) {
```

Comment:
❓ Did this `Constants.blazeScreenTransitionsDelay ` usage get mixed up? It should be used in `navigateToBlazeCampaignDetails` right?

Reply:
Great catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/13843/commits/05ac3d3c12b857c84f6654b3932735e86a61d7eb